docs/faq.md

```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## My fuzzer stopped finding new coverage/bugs after a while.
```

Comment:
Phrase as a question?

What if my fuzzer does not find new coverage or bugs after a while.